### PR TITLE
Draft: Re-connect to network on visibility change

### PR DESCRIPTION
**Pull request by schmidt-sebastian**
Hey there! So you want to contribute to a Firebase SDK? 
Before you file this pull request, please read these guidelines:

### Discussion

  * Read the contribution guidelines (CONTRIBUTING.md).
  * If this has been discussed in an issue, make sure to link to the issue here. 
    If not, go file an issue about this **before creating a pull request** to discuss.

### Testing

  * Make sure all existing tests in the repository pass after your change.
  * If you fixed a bug or added a feature, add a new test to cover your code.

### API Changes

  * At this time we cannot accept changes that affect the public API.  If you'd like to help 
    us make Firebase APIs better, please propose your change in an issue so that we 
    can discuss it together.

### CODE REVIEW

File: packages/firestore/src/platform/browser/connectivity_monitor.ts [hunk 1]
```diff
@@ -48,11 +48,25 @@ export class BrowserConnectivityMonitor implements ConnectivityMonitor {
   shutdown(): void {
     window.removeEventListener('online', this.networkAvailableListener);
     window.removeEventListener('offline', this.networkUnavailableListener);
+    window.addEventListener('visibility', this.visibilityChangeListener);
```
Comment (schmidt-sebastian) [inline packages/firestore/src/platform/browser/connectivity_monitor.ts:51]: s/add/remove

**Comment by schmidt-sebastian** [comment]
@rafikhan Can you move this into your own branch? I am going to close this PR as it shows up in my list of TODO items.

**Comment by rafikhan** [comment]
> @rafikhan Can you move this into your own branch? I am going to close this PR as it shows up in my list of TODO items.

Done.  It's at `khanrafi/cm`

**Comment by ehsannas** [comment]
why did this PR get closed?

**Pull request closed without merging**